avm/res/web/serverfarm/README.md

```diff
@@ -54,7 +54,7 @@ module serverfarm 'br/public:avm/res/web/serverfarm:<version>' = {
       tier: 'Premium'
     }
     // Non-required parameters
-    location: '<location>'
+    location: 'eastus'
```

Comment:
Needs updating

Reply:
This is generated from the test files what needs updating here?

Reply:
Just testing you!